### PR TITLE
update to node-ical, handle MS timezones and timezones in general in cal entries

### PR DESCRIPTION
**Pull request by sdetweil**
- Does the pull request solve a **related** issue?
#1798 
#2085

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,10 +75,10 @@
 		"express-ipfilter": "^1.1.2",
 		"feedme": "^1.2.0",
 		"helmet": "^3.23.3",
-		"ical": "^0.8.0",
-		"iconv-lite": "^0.6.2",
+		"iconv-lite": "latest",
```
Comment (rejas) [inline package.json:78]: On purpose or accidentally reverted?

File: package.json [hunk 2]
```diff
@@ -75,10 +75,10 @@
...
 		"module-alias": "^2.2.2",
-		"moment": "^2.28.0",
+		"moment": "latest",
```
Comment (rejas) [inline package.json:80]: See above

File: modules/default/calendar/calendarfetcher.js [hunk 3]
```diff
@@ -5,7 +5,7 @@
  * MIT Licensed.
  */
 const Log = require("../../../js/logger.js");
-const ical = require("ical");
+const ical = require("node-ical");
```
Comment (rejas) [inline modules/default/calendar/calendarfetcher.js:8]: Are there any hacks that we can remove now?

File: package.json (hunk 2 above)
Comment (sdetweil) [reply package.json:80]: probably accidentally. 

remove ical, add node-ical  was all I was trying to do

I don't think any hacks can be removed.. I think its pretty clean now

**Comment by hangorazvan** [comment]
great job:
`Cannot find module 'ical'`

**Comment by sdetweil** [comment]
where are my comments?  I resolved this yesterday

File: package.json (hunk 1 above)
Comment (sdetweil) [reply package.json:78]: accidentally I said yesterday

**Pull request merged**